openfisca_aotearoa/entities.py

```diff
@@ -3,38 +3,34 @@
 # This file defines the entities needed by our legislation.
 from openfisca_core.entities import build_entity
 
-Household = build_entity(
-    key = "household",
-    plural = "households",
-    label = u'Household',
-    doc = '''
-    Household is an example of a group entity.
-    A group entity contains one or more individual·s.
-    Each individual in a group entity has a role (e.g. parent or children). Some roles can only be held by a limited number of individuals (e.g. a 'first_parent' can only be held by one individual), while others can have an unlimited number of individuals (e.g. 'children').
 
-    Example:
-    Housing variables (e.g. housing_tax') are usually defined for a group entity such as 'Household'.
+# Using middle English spelling due to `property` being a Python keyword.
+Propertee = build_entity(
+    key = "propertee",
+    plural = "properties",
+    label = u'Property',
+    doc = '''
+    A Property represents accommodation that is owned by a group of Persons.
 
-    Usage:
-    Check the number of individuals of a specific role (e.g. check if there is a 'second_parent' with household.nb_persons(Household.SECOND_PARENT)).
-    Calculate a variable applied to each individual of the group entity (e.g. calculate the 'salary' of each member of the 'Household' with salaries = household.members('salary', period = MONTH); sum_salaries = household.sum(salaries)).
+    Example usage:
+    Check the number of individuals of a specific role: check how many persons co-own the property: `properties.nb_persons(Propertee.OWNER)`.
+    Calculate a variable applied to each tenant of the group entity: calculate the income of each member of the Property: `tenants_incomes = properties.members('income', period = MONTH); tenants_total_income = properties.sum(tenants_incomes)`.
 
-    For more information, see: http://openfisca.org/doc/coding-the-legislation/50_entities.html
+    For more information on group entities, see: http://openfisca.org/doc/coding-the-legislation/50_entities.html
     ''',
     roles = [
         {
-            'key': 'parent',
-            'plural': 'parents',
-            'label': u'Parents',
+            'key': 'owner',
+            'plural': 'owners',
+            'label': u'Owners',
             'max': 2,
```

Comment:
This limitation seems very arbitrary. Is there anything in law that limits how many owners there can be of an estate?  🙂